src/csharp/Microsoft.Spark.E2ETest/IpcTests/BroadcastTests.cs

```diff
@@ -34,9 +34,12 @@ public BroadcastTests(SparkFixture fixture)
         /// <summary>
         /// Test Broadcast support by using multiple broadcast variables in a UDF.
         /// </summary>
-        [Fact]
-        public void TestMultipleBroadcastWithoutEncryption()
+        [Theory]
+        [InlineData("true")]
+        [InlineData("false")]
+        public void TestMultipleBroadcast(string isEncryptionEnabled)
         {
+            _spark.SparkContext.GetConf().Set("spark.io.encryption.enabled", isEncryptionEnabled);
```

Comment:
We need something like `withSQLConf` in Scala `sql_conf` in PySpark to restore the config value. @Niharikadutta Could you file an issue so we can follow up? Thanks!

Reply:
I'm sorry I don't think I understood this. What is meant by restoring the config value? Is there an existing function like that in PySpark?

Reply:
yes: https://github.com/apache/spark/blob/master/python/pyspark/testing/sqlutils.py#L167

Reply:
This is going back to the comment regarding adding a Dispose method, but maybe adding a more generic method that all tests can reuse.

Reply:
Understood. Created #693 . Thanks!